### ISSUE TITLE
Plugin throws exception when importing a new organism 

### ISSUE THREAD

**Issue by chrtannus** [GeneMANIA/genemania#204]
Try to import an organism from a text file:

```
java.lang.NullPointerException
    at org.genemania.domain.GeneNamingSource$$EnhancerByCGLIB$$974367c1.getRank(<generated>)
    at org.genemania.plugin.NetworkUtils.getPreferredGene(NetworkUtils.java:137)
    at org.genemania.plugin.completion.PasteResult.addSynonym(PasteResult.java:97)
    at org.genemania.plugin.completion.CompletionTransferHandler.validate(CompletionTransferHandler.java:252)
    at org.genemania.plugin.completion.CompletionTransferHandler.access$000(CompletionTransferHandler.java:65)
    at org.genemania.plugin.completion.CompletionTransferHandler$1.runTask(CompletionTransferHandler.java:95)
    at org.genemania.plugin.task.GeneManiaTask.run(GeneManiaTask.java:45)
    at org.genemania.plugin.task.TaskDispatcher$TaskHandler.run(TaskDispatcher.java:113)
    at org.genemania.plugin.cytoscape3.OsgiTaskHandler.runInternal(OsgiTaskHandler.java:43)
    at org.genemania.plugin.cytoscape3.OsgiTaskHandler.access$000(OsgiTaskHandler.java:26)
    at org.genemania.plugin.cytoscape3.OsgiTaskHandler$1.runWithHack(OsgiTaskHandler.java:37)
    at org.genemania.plugin.cytoscape3.ThreadContextClassLoaderHack.run(ThreadContextClassLoaderHack.java:41)
    at org.genemania.plugin.cytoscape3.OsgiTaskHandler.run(OsgiTaskHandler.java:39)
    at java.base/java.lang.Thread.run(Thread.java:834)
```

<img width="50%" alt="screenshot" src="https://github.com/GeneMANIA/genemania/assets/989686/716dd501-f1bb-4a4b-9c5d-6e2a9e6de2a8">

**Sample:**
[_aThaliana_identifier_mappings (SMALL-TEST).txt](https://github.com/GeneMANIA/genemania/files/11737699/_aThaliana_identifier_mappings.SMALL-TEST.txt)

**Comment by chrtannus** [comment]
The problem is that to deserialize the gene data from _Lucene_, we use the **cglib** library for lazy loading, since the domain objects (_commons_ package) contains many circular dependencies.

So looking at the cglib project page, It states:

_**IMPORTANT NOTE:** cglib is unmaintained and does not work well (or possibly at all?) in newer JDKs, particularly JDK17+. If you need to support newer JDKs, we will accept well-tested well-thought-out patches... but you'll probably have better luck migrating to something like ByteBuddy._

https://github.com/cglib/cglib

**Comment by chrtannus** [comment]
Fixed in version `3.5.3`.
